### PR TITLE
Added lerna

### PR DESCRIPTION
**Pull request by anton-nagornyi**

### CODE REVIEW

**Comment by JustFly1984** [comment]
Well, it is failing, cos we don't have PR branch deployment, so don't bother about it.

**Comment by JustFly1984** [comment]
Sorry I don't have energy to go through commit today, will try to find the time tomorrow. @uriklar maybe you can find a time to review?

**Comment by anton-nagornyi** [comment]
@JustFly1984 Checkout CONTRIBUTING.md

@uriklar 
We started to use lerna because we wanted to get rid of the peer dependency hell. Storybook is conflicting with docz on @emotion/core dependency. Now those projects are separated and conflicts are resolved. Also I think it is a good idea to keep things separate )

File: packages/react-google-maps-api/src/components/streetview/StreetViewPanorama.js [hunk 1]
```diff
@@ -73,13 +73,12 @@ export class StreetViewPanorama extends PureComponent {
 
   componentDidMount = () => {
     const streetViewPanorama = this.context.getStreetView()
-
     this.setState(
       () => ({
         streetViewPanorama
       }),
       () => {
-        this.state.streetViewPanorama.setVisible(true)
```
Comment (uriklar) [inline packages/react-google-maps-api/src/components/streetview/StreetViewPanorama.js]: Is this change intentional?

File: packages/docz/src/Marker.mdx [hunk 2]
```diff
@@ -2,11 +2,7 @@
 name: Marker
 ---
 
-import {
-  GoogleMap,
-  LoadScript,
-  Marker
-} from "..";
+import { GoogleMap, LoadScript, Marker } from "react-google-maps-api/lib";
```
Comment (uriklar) [inline packages/docz/src/Marker.mdx]: Why are we using the /lib here?

File: packages/react-google-maps-api/src/components/streetview/StreetViewPanorama.js (hunk 1 above)
Comment (anton-nagornyi) [reply packages/react-google-maps-api/src/components/streetview/StreetViewPanorama.js]: Not really. StreetViewPanorama is not finished yet and is not available for use. I made some changes to it before lerna and they went to PR accidentally
